plugins/generic/usageEvent/UsageEventPlugin.inc.php

```diff
@@ -286,6 +286,18 @@ function _buildUsageEvent($hookName, $args) {
 			$request, null, $canonicalUrlPage, $canonicalUrlOp, $canonicalUrlParams
 		);
 
+		// Make sure we log the server name and not aliases.
+		$configBaseUrl = Config::getVar('general', 'base_url');
+		$requestBaseUrl = $request->getBaseUrl();
+		if ($requestBaseUrl !== $configBaseUrl) {
+			// Make sure it's not an url override (no alias on that case).
+			if (!in_array($requestBaseUrl, Config::getContextBaseUrls()) && 
+					$requestBaseUrl !== Config::getVar('general', 'base_url[index]')) {
```

Comment:
Here's the scenario I'm considering:

config.inc.php
base_url = http://scholarlyexchange.org/ojs
base_url[index] = http://scholarlyexchange.org/ojs
base_url[myJournal] = http://myjournal.tld

Apache:
ServerName myjournal.tld
ServerAlias www.myjornal.tld
ServerAlias myjournal.myorg.tld

HTTP_HOST = http://www.myjournal.tld
or
HTTP_HOST = http://myjournal.myord.tld